### DISCUSSION TITLE
change sex

### DISCUSSION THREAD

**Discussion by mariannevanharten** [FamilySearch/GEDCOM#521] [Ideas]
Some people changed gender, for example in 2020. Most genealogy software assigns the new gender to all facts, while some facts the new gender took place at the time of the original birth gender.
#A is born a boy, grows up to be a man, marries girl #B. Years later, #B dies, after which #A discovers she is a woman. The marriage of #A and #B is and remains a marriage between man and woman. #A may not be referred to as a woman here.
I suggest that the gedcom tag SEX accepts underlying tags, including DATE, so that the correct name and gender of #A can be displayed based on the date.
#A had another brief relationship with girl #C well before marrying #B. So this relationship too was a relationship between a man and a woman.

**Comment by tychonievich** [comment]
[The definition of `g7:enumset-SEX`](https://gedcom.io/specifications/FamilySearchGEDCOMv7.html#enumset-SEX) includes this:

> This can describe an individual’s reproductive or sexual anatomy at birth. Related concepts of gender identity or sexual preference are not currently given their own tag. Cultural or personal gender preference may be indicated using the FACT tag.

The decision to make `SEX` be reproductive sex long predates my involvement in GEDCOM, possibly motivated by the use of GEDCOM for heredity and other biological contexts. Regardless of its origin, there are multiple applications that actively depend on this structure having a reproductive biology meaning to facilitate DNA and heredity uses; changing it to a different concept would be a breaking change for those applications.

Adding a gender identity structure (separate from `SEX`) is something that has been discussed multiple times. The main challenge I see to its implementation is in the deails, not the broad idea. Most notably, what values it should have? There have been proponents for just three values (male, female, other) or for more than a dozen (e.g. [#494.5](https://github.com/FamilySearch/GEDCOM/issues/494#issuecomment-2198200637)) or for making it a free-text field instead of an enumerated value. I see pros and cons with each and don't know how to pick.

I'd love to know if any existing applications have an implementation of gender attributes that we could make standard in the next version of GEDCOM.

**Comment by jkr-wrk** [comment]
### History != Biological
#### Legal Registrations:
-   Historically, individuals born with the same biological reproduction organs could be legally registered as either a man or a woman in all legal documents.
-  These individuals could also be registered as having biological children, regardless of their actual biological contribution.

#### Family Manipulations:
- Legal documents could be manipulated to reflect non-biological relationships:
.. A woman could get pregnant and register a different man as the biological father.
.. Married couples could secretly adopt children and register them as their biological offspring.

#### Why GEDCOM Reflects Legal Structures

. Historically, family trees created using the GEDCOM format have always mirrored the legal family structure rather than the DNA relationships.
. This is because, until recently, there was no reliable method to verify biological relationships.
. The only way to accurately determine the biological parentage of a child is through DNA research, which has only become widely available in recent years.

#### Recent Developments

Acceptance of Diverse Family Structures:
. Society has increasingly accepted that family structures can differ from traditional norms.
. There is a growing willingness to register these varied family structures openly and accurately.

### Proposal for GEDCOM Format

Adaptation to Include DNA Relations:
. It would be beneficial to adapt the GEDCOM format to include both 'DNA' relationships and the family structures that are legally registered or lived by.
. This dual approach allows for a more comprehensive and accurate representation of family histories.

#### Flexibility for Users:
. The person creating the GEDCOM file should have the flexibility to decide which type of relationship—legal or DNA—they want to visualize.
. This flexibility ensures that both the legal family structure and the biological DNA relationships can be accurately documented.

#### Benefits of Legal Structure Availability

Enhanced Accuracy:
. By making the legal structure available, users can cross-reference and verify the DNA structure more accurately.
. This ensures that any discrepancies between legal documentation and biological relationships can be identified and documented.

Representation of Modern Family Structures:
. If someone wants to register and visualize a modern family structure, such as Male-Male+children or a remarried Male who transitioned to Female in the second marriage, it is better to register the actual situation.
. This approach provides a more truthful and comprehensive representation of family histories, rather than misrepresenting the family structure as a clean DNA relationship.

And it would not break historical data, because that is based on legal assumptions already and not biological accuracy.

**Comment by mariannevanharten** [comment]
In my family tree application (Webtrees) I am using a module to create a family tree overview, a story of the descendants of a given person. I create a overview of the descendants of the grandfather of B.
In my above mentioned example person A was called John and now she is called Jane. Jane's record shows her as woman, The name John is also recorded and hidden. Currently there is a fact "sex at birth".
The story created by the module now shows:
B married Jane. She is born in .... at ....
but that's wrong; it should be:
B married John. He is born in ... at ...

I prefer to use in my Gedcom-record:
0 @X1@ INDI
1 NAME Jane //
2 SEX F
2 DATE FROM 2020
1 NAME John //
2 SEX B
3 DATE TO 2020
1 FAMS @X3@
0 @X2@ INDI
1 NAME B //
2 SEX F
0 @X3@ FAM
1 HUSB @X1@
1 WIFE @X2@
1 MARR
2 DATE 2010

After using the module I edited the output to achieve the right text.

**Comment by dthaler** [comment]
Discussion in GEDCOM Steering committee meeting:
* A gender change could be an _event_ with a date, whether `EVEN` today or an extension or a new tag in a future version of GEDCOM.
* Keeping `SEX` as the sex assigned at birth is essential for backwards compatibility, but a different tag could be used for gender (as an _attribute_) as of other dates or date ranges, such as a `FACT` (as the GEDCOM spec suggests in the quote Luther gave above) an extension like `_GENDER` or a standard tag in a future version of GEDCOM.  Such an attribute could have an enumerated value, but the set of possible values is not yet clear (specifically which set to use), so there hasn't been a proposal yet.  We would like to see experience from existing use before standardizing which values to use.

**Comment by jcnventura** [comment]
@mariannevanharten taking into account your reply and the accepted fact that sex, gender and sexuality are 3 different concepts with sex being (almost universally) accepted as referring to the biological sex (i.e. does a genetically healthy individual have XX or XY chromosomes), maybe the best title for this discussion should be "gender change"?

What happened to individual #A was a gender change, not a sex change (as long as we can agree to the definition of sex being different from that of gender, and sex being the biological sex). Until the time where DNA editing makes it possible for an individual to change their chromosomes (and survive the process), a sex change is simply not possible yet.

**Reply by Norwegian-Sardines** [reply]
Some individuals can have XXY, XYY and even XXX and XXYY chromosomes as well.

Some individuals are born looking like one sex, but really have a genetically different sex!

What are the boundaries of “Gender Change” does this include only dressing as a different gender? Or changing their body to look like a different gender?  

I’m not comfortable making broad design statements without input from the community this affects.

**Reply by jcnventura** [reply]
> Some individuals can have XXY, XYY and even XXX and XXYY chromosomes as well

Yes, I know. I tried to simplify not referring to those (and probably others) as "genetically healthy individual have XX or XY chromosomes". I didn't express myself very well. What I meant was that biological sex --usually-- indicates either XY (M), XX (F) but there are many other cases. These other cases are usually given syndrome names as they sometimes carry health complications.
The discussion of what sex to assign to individuals with these chromosome abnormalities is way beyond a Github discussion, but thankfully the standard already provides the X value for cases that fall outside the F and M. And to clarify, I'm not saying that only XY is M and only XX is F, XXX are usually also perfectly normal females (even if they do have some health complications).

**Reply by jcnventura** [reply]
> What are the boundaries of “Gender Change”

There are no boundaries. The individual decides its gender. It can decide to say they are now a woman or man or non-binary, and retain the exact same appearance as before. Gender lives in the head of the individual, and may or may not lead to external changes.

**Reply by Norwegian-Sardines** [reply]
When I asked about boundaries is it a “Gender Change” for an individual whom the doctor looked down saw no penis and said female, but in reality the individual was XY?  They then later in life needed an operation to present as a male.  Is this still a “Gender Change” or a “Sex Change”!  This is not in their head!  

Most of this discussion is outside the needs of this thread and why GEDCOM has a tough time with these issues!  The answer is not as straightforward as people make it!  I’m done!
